docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -43,7 +43,7 @@ A short description for the feature. <2>
 . Do this second:
 ----
 <1> {walkthrough-name} title, you can only have one level 1 heading per file
-<2> The first line of the introduction is displayed as a tag line for the {walkthrough-name}.
+<2> The first line of the introduction is displayed as a tag line for the {walkthrough-name}.  Please note that attributes will not be rendered in this tag line.
```

Comment:
```suggestion
<2> The first line of the introduction is displayed as a tag line for the {walkthrough-name} and displayed on the Solution Explorer home page.  
Do not use AsciiDoc attributes in this line because they are not displayed correctly on the home page.
```